### PR TITLE
Fix MiniWindow Hiding Error

### LINKED ISSUE #51: MiniPlayer will not be hidden correctly
How to reproduce:

1. Launch player
2. Switch to mini mode
3. Drag mini window to margin and hide it
4. Switch to main mode
5. Switch to mini mode

Behavior:

Hide when `MouseMove`(Hover)
Show when `MouseLeave`

### PR DESCRIPTION
**Pull request by Kyrios0**
fix #51

### CODE REVIEW

File: OsuPlayer.Wpf/Windows/MiniWindow.xaml.cs [hunk 1]
```diff
@@ -26,7 +26,7 @@ public partial class MiniWindow : WindowEx
         private double _screenWidth;
         private double _screenHeight;
 
-        private Rectangle _currentArea;
+        private static Rectangle _currentArea;
```
Comment (Milkitic) [inline OsuPlayer.Wpf/Windows/MiniWindow.xaml.cs:29]: `static` shouldn't be here. The default value can be initialized from `Loaded` event

File: OsuPlayer.Wpf/Windows/MiniWindow.xaml.cs [hunk 2]
```diff
@@ -93,8 +95,16 @@ private void Window_Loaded(object sender, RoutedEventArgs e)
                 Left = _screenWidth - ActualWidth - 20;
                 Top = _screenHeight - ActualHeight - 100;
             }
+            if (area != null && area.Length == 4)
+            {
+                _currentArea = new Rectangle(area[0], area[1], area[2], area[3]);
+            }
```
Comment (Milkitic) [inline OsuPlayer.Wpf/Windows/MiniWindow.xaml.cs:101]: Add an `else` here and initialize a default value: 
```cs
_currentArea = Screen.GetWorkingArea(new System.Drawing.Point((int)Left, (int)Top));
```

**Review by Milkitic — CHANGES_REQUESTED** [review]

File: OsuPlayer.Wpf/Windows/MiniWindow.xaml.cs (hunk 1 above)
Comment (Kyrios0) [reply OsuPlayer.Wpf/Windows/MiniWindow.xaml.cs]: That's my fault, fixed 5c1f431.

File: OsuPlayer.Wpf/Windows/MiniWindow.xaml.cs (hunk 2 above)
Comment (Kyrios0) [reply OsuPlayer.Wpf/Windows/MiniWindow.xaml.cs:101]: added 5c1f431

**Review by Milkitic — APPROVED** [review]

**Pull request merged**